mantidimaging/core/utility/version_check.py

```diff
@@ -5,92 +5,153 @@
+            # just returns as "unstable" label
+            self._conda_installed_version = ""
+            self._conda_installed_label = "unstable"
+            return
+
+        self._conda_installed_version, self._conda_installed_label = local_package.split()
+        self._conda_installed_label = self._conda_installed_label.rpartition("/")[2]
+
+    def _retrieve_conda_available_version(self) -> None:
+        try:
+            response = requests.get("https://api.anaconda.org/package/mantid/mantidimaging")
+            remote_main_version = json.loads(response.content)["latest_version"]
+            remote_unstable_version = json.loads(
+                response.content)["versions"][-1] if len(json.loads(response.content)["versions"]) > 0 else ''
+        except Exception:
+            # if anything goes wrong, in the end we don't have the version
+            LOG.info("Could not connect to Anaconda remote to retrieve the latest version")
+            self._conda_available_version = ""
+            return
+
+        if self.get_conda_installed_label() == "main":
+            self._conda_available_version = remote_main_version
+        else:
+            self._conda_available_version = remote_unstable_version
+
+
+versions = CheckVersion()
+
+
+def _parse_version(package_version_string: Optional[str]) -> ParsedVersion:
+    if package_version_string is None:
+        raise ValueError
     local_version, local_commits_since_last = package_version_string.split("_")
     return ParsedVersion(tuple(map(int, local_version.split("."))), int(local_commits_since_last))
 
 
-def _do_version_check(local: ParsedVersion, remote: ParsedVersion, action: Callable[[str], None], is_main_label: bool):
-    if local.version < remote.version or local.commits < remote.commits:
-        # no suffix if main, else adds Unstable
-        suffix = "" if is_main_label else " Unstable"
-
-        msg = f"Not running the latest Mantid Imaging{suffix}. Found version {_make_version_str(local)}, " \
-              f"latest: {_make_version_str(remote)}. Please check the terminal for an update command!"
-        LOG.info(msg)
-
-        # for unstable packages these run variables are prepended to the command
-        # for main packages nothing is prepended, and the script's defaults are used
-        command_prefix = "" if is_main_label else "ENVIRONMENT_NAME=mantidimaging_unstable REPO_LABEL=unstable "
-
-        LOG.info("To update your environment please copy and run the following command:\n\n"
-                 "source /opt/miniconda/bin/activate /opt/miniconda && "
-                 f"{command_prefix}source "
-                 "<(curl -s https://raw.githubusercontent.com/mantidproject/mantidimaging/master/install.sh)")
-        action(msg)
-    else:
-        LOG.info("Running the latest Mantid Imaging")
-
-
 def _make_version_str(parsed: ParsedVersion) -> str:
     return f"{'.'.join([str(v) for v in parsed.version])}_{parsed.commits}"
```

Comment:
I don't think this function is used anymore (only in a test, but we can remove both)

Reply:
Removed in b90e47e